README.md

```diff
@@ -1,3 +1,7 @@
+### Disclaimer
```

Comment:
You can emphsise the block with
> [!NOTE]
> ...

as done in https://github.com/ClickHouse/copier/commit/0b899f642e7a019aa829425b78aeace1b47f2bd0